src/inc-lib-init.php

```diff
@@ -2837,6 +2837,17 @@ function lovd_queryError ($sErrorCode, $sSQL, $sSQLError, $bHalt = true)
 
 
 
+function lovd_removeRefSeq ($sVariant)
+{
+    // This function removes the reference sequence from
+    //  a variant description.
+    return substr(strstr($sVariant, ':'), 1);
+}
```

Comment:
Do you think this function should do something "special" when the input has no `:`?

The "downside" of creating such a function is that you may need input validation, especially when notices or warnings could be generated by providing invalid input. In this case, sending an empty string or a boolean doesn't seem to cause problems. But sending in an array does. Better check if `$sVariant` is a string before doing anything? Also, for instance, do you think it's acceptable that running `lovd_removeRefSeq()` on `c.100del` returns an empty string?

You wanted a function, these are the consequences :smiling_imp: